core/templates/components/summary-tile/exploration-summary-tile.component.ts

```diff
@@ -139,20 +147,23 @@ export class ExplorationSummaryTileComponent implements OnInit, OnDestroy {
     this.windowRef.nativeWindow.location.href = this.getExplorationLink();
   }
 
-  getAverageRating(): number {
-    if (!this.getRatings) {
-      return null;
+  // Function will return undefined when Exploration Ratings are not present.
+  getAverageRating(): number | undefined {
```

Comment:
Should we return `null` instead?

Reply:
`.computeAverageRating` can return an `undefined` value that's why made the return type to be `number | undefined` else if we would want to return null, would've to make it `number | null | undefined` . I think just `number | undefined` will be better?

Reply:
Can we modify `computeAverageRating` to return `null` too?

Reply:
Done